### PR TITLE
New not found page

### PR DESCRIPTION
**Pull request by Michal170**

### CODE REVIEW

File: harcownik-frontend/src/components/notfound/NotFound.jsx [hunk 1]
```diff
@@ -2,11 +2,23 @@ import React from 'react'
 
 const NotFound = () => {
     return (
-        <div>
-            <h1>
-                Not Found
-            </h1>
-        </div>
+                <div>
+                  <meta charSet="UTF-8" />
```
Comment (pawel150199) [inline harcownik-frontend/src/components/notfound/NotFound.jsx:6]: Not needed. We have things like this in index.js file.

File: harcownik-frontend/src/components/notfound/NotFound.jsx [hunk 2]
```diff
@@ -2,11 +2,23 @@ import React from 'react'
...
+                  <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
```
Comment (pawel150199) [inline harcownik-frontend/src/components/notfound/NotFound.jsx:7]: the same like previous

File: harcownik-frontend/src/components/notfound/NotFound.jsx [hunk 3]
```diff
@@ -2,11 +2,23 @@ import React from 'react'
...
+                  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (pawel150199) [inline harcownik-frontend/src/components/notfound/NotFound.jsx:8]: same

File: harcownik-frontend/src/components/notfound/NotFound.jsx [hunk 4]
```diff
@@ -2,11 +2,23 @@ import React from 'react'
...
+                  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" />
```
Comment (pawel150199) [inline harcownik-frontend/src/components/notfound/NotFound.jsx:9]: same

File: harcownik-frontend/src/components/notfound/NotFound.jsx [hunk 5]
```diff
@@ -2,11 +2,23 @@ import React from 'react'
...
+                      <h1 className="display-1 fw-bold">404</h1>
+                      <p className="fs-2"> <span className="text-danger">Opps!</span> Strona nie została znaleziona.</p>
+                      <p className="lead">
+                        Strona której szukasz nie istnieje.
+                      </p>
+                      <a href="/" className="btn btn-primary">Wróć do strony głównej</a>
+                    </div>
+                  </div>
+                </div>
+
```
Comment (pawel150199) [inline harcownik-frontend/src/components/notfound/NotFound.jsx:21]: finally it should look like following code:

import React from 'react'

const NotFound = () => {
    return (
        <div className="d-flex align-items-center justify-content-center vh-100">
            <div className="text-center">
                <h1 className="display-1 fw-bold">404</h1>
                <p className="fs-2"> <span className="text-danger">Opps!</span> Strona nie została znaleziona.</p>
                <p className="lead">
                    Strona której szukasz nie istnieje.
                </p>
                <a href="/" className="btn btn-primary">Wróć do strony głównej</a>
            </div>
        </div>  
    );
}

export default NotFound;

File: harcownik-frontend/src/components/notfound/NotFound.jsx (hunk 4 above)
Comment (pawel150199) [inline harcownik-frontend/src/components/notfound/NotFound.jsx]: Not needed. We have things like this in index.js file.

File: harcownik-frontend/src/components/notfound/NotFound.jsx [hunk 6]
```diff
@@ -2,11 +2,23 @@ import React from 'react'
...
+                      <p className="lead">
+                        Strona której szukasz nie istnieje.
+                      </p>
+                      <a href="/" className="btn btn-primary">Wróć do strony głównej</a>
+                    </div>
+                  </div>
+                </div>
+              
     );
 }
```
Comment (pawel150199) [inline harcownik-frontend/src/components/notfound/NotFound.jsx]: finally it should look like following code:

```javascript
import React from 'react'

const NotFound = () => {
    return (
        <div className="d-flex align-items-center justify-content-center vh-100">
            <div className="text-center">
                <h1 className="display-1 fw-bold">404</h1>
                <p className="fs-2"> <span className="text-danger">Opps!</span> Strona nie została znaleziona.</p>
                <p className="lead">
                    Strona której szukasz nie istnieje.
                </p>
                <a href="/" className="btn btn-primary">Wróć do strony głównej</a>
            </div>
        </div>  
    );
}

export default NotFound;
```

**Review by Kduszynska — APPROVED** [review]
looks good to me

**Pull request merged**